tasks/extract-binaries-from-image/tests/test-extract-binaries-multiple-components-desired-only.yaml

```diff
@@ -0,0 +1,135 @@
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: test-extract-binaries-multiple-components-desired-only
+spec:
+  description: |
+    Run the extract-binaries-from-image task with three components in the snapshot
+    but only extract binaries for the desired components. Result should be 2.
+  workspaces:
+    - name: tests-workspace
+  tasks:
+    - name: setup
+      workspaces:
+        - name: data
+          workspace: tests-workspace
+      taskSpec:
+        workspaces:
+          - name: data
+        steps:
+          - name: setup-snapshot
+            image: quay.io/redhat-appstudio/release-service-utils:bc81bfed6062a386e48a76b252c6f33b52c411b0
```

Comment:
Can you update this please? We recently migrated to the konflux-ci org in quay, and as part of that I updated every ref in this repo. If you rebase, you should see the new image in the task file